src/outputs/parthenon_hdf5.cpp

```diff
@@ -890,8 +888,8 @@ void PHDF5Output::WriteOutputFileImpl(Mesh *pm, ParameterInput *pin, SimTime *tm
       if (!is_allocated) {
         if (vinfo.is_sparse) {
           hsize_t N = varSize * vlen;
-          memset(tmpData.data() + index, 0, N * sizeof(OutT));
-          index += N;
+          for (int i = 0; i < N; ++i)
+            tmpData[index++] = std::numeric_limits<Real>::quiet_NaN();
```

Comment:
is there a reason this couldn't be `memset` to `quiet_NaN`? It probably doesn't matter.

Reply:
yeah, `memset` might be better but I don't think it matters. This should probably be switched back to zero, since previously the non-allocated sparse fields were zero.